tpu/config.txt

```diff
@@ -0,0 +1,3 @@
+BASE_IMAGE_TAG=v108
+LIBTPU_IMAGE_TAG=libtpu_1.1.0_RC00
+TENSORFLOW_WHL_IMAGE_TAG=local
```

Comment:
Should this be the image build by this pipeline?

Reply:
Yes, cleaned this up.